src/Tokenizer/Tokens.php

```diff
@@ -156,19 +156,21 @@ public static function fromArray($array, $saveIndexes = null)
      */
     public static function fromCode($code)
     {
-        $codeHash = self::calculateCodeHash($code);
+        if ('' !== $code) {
```

Comment:
this change is not tested anywhere.
----------

what is the user case - why we need this change?
```php
$a = Tokens::fromCode('');
$b = Tokens::fromCode('');
```

on master, `$a === $b`, on your branch, they don't. what is a benefit of this change? It's an exception of how the collection is reused.

also, if one wants to have a new, empty collection - there is perfect way to do it - `new Tokens()`